lib/inv_cg_quda.cpp

```diff
@@ -242,6 +244,7 @@ namespace quda {
 
     // whether to select alternative reliable updates
     bool alternative_reliable = param.use_alternative_reliable;
+    bool advanced_feature = !(param.precondition_no_advanced_feature && param.is_preconditioner);
```

Comment:
As much as I love the name `advanced_feature`, it's not very descriptive what it does 😉

Reply:
https://github.com/lattice/quda/pull/1061/commits/cefb2a527fa7cb1303a9e7487b9fea4f39d1bf8d